app/repositories/SessionCacheRepository.scala

```diff
@@ -65,7 +65,7 @@ class DefaultSessionCacheRepository @Inject()(mongoComponent: MongoComponent,
   override def getAccountLinks(sessionId: String): Future[Option[Seq[AccountLink]]] = {
     for {
       record <- collection.find(equal("_id", sessionId)).toSingle().toFutureOption()
-      result = record.flatMap(a => Option(a.accountLinks.filter(link => link.accountNumber.size == 7)))
+      result = record.flatMap(a => Option(a.accountLinks.filter(link => link.accountNumber.length == 7)))
```

Comment:
7 is not generating the magic number warning but still would be good to make a meaningful constant for it with in the method. As it will be safe to update the value in future (if we need)